src/main/java/core/basesyntax/Main.java

```diff
@@ -0,0 +1,18 @@
+package core.basesyntax;
+
+public class Main {
+    public static void main(String[] args) {
+        FigureSupplier figureSupplier = new FigureSupplier();
+        Figure[] figures = new Figure[6];
+
+        for (int i = 0; i < figures.length; i++) {
+            if (i < (figures.length / 2)) {
+                figures[i] = figureSupplier.getRandomFigure();
+                figures[i].toDrawFigure();
+            } else {
+                figures[i] = figureSupplier.getDefaultFigure();
+                figures[i].toDrawFigure();
```

Comment:
Similarly to the previous comment, consider moving the drawing logic outside of the loop.